core/src/main/java/org/elasticsearch/transport/TransportRequestOptions.java

```diff
@@ -92,4 +44,50 @@ public boolean compress() {
     public Type type() {
         return this.type;
     }
+
+    public static final TransportRequestOptions EMPTY = new TransportRequestOptions.Builder().build();
```

Comment:
Seems strange to have no only a `builder()` method, but also be able to do `new TransportRequestOptions.Builder()` to me